collect_app/src/main/java/org/odk/collect/android/fragments/AppListFragment.java

```diff
@@ -60,7 +62,8 @@
     protected SimpleCursorAdapter mListAdapter;
     protected LinkedHashSet<Long> mSelectedInstances = new LinkedHashSet<>();
     protected EditText mInputSearch;
-
+    private int moldSelected;
```

Comment:
We have called both files, one in all activities and other in 'delete saved form fragment'. Thats why.